lib/pages/home/home_header.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+
+class HomeHeader extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Text(
+          // TODO
+          'MONDAY, JUNE 15',
+          style: Theme.of(context).textTheme.subtitle1!.copyWith(
+                fontSize: 13,
+              ),
+        ),
+        SizedBox(height: 5.0),
+        Row(
+          children: [
+            Expanded(
+              child: Text(
+                // TODO
```

Comment:
better put a TODO on the top because you will receive a notifier at once for all the widget render

Reply:
The only 1 wip thing here is the text value so I would like to keep the TODO comment exactly where it should be 🤓